pages/about-us.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Navbar from './../Components/navbar.js'
+import Body from './../Components/body.js'
+import { createMuiTheme, makeStyles, ThemeProvider } from '@material-ui/core/styles';
```

Comment:
Do we need this? Does it get used anywhere?

Reply:
Just put these here as a placeholder copied from the home page. Navbar will remain since we want that on all pages, but we should probably have a separate body component for each page.